plugins/editor/src/editor/Editor.cpp

```diff
@@ -1067,9 +1067,9 @@ void Editor::Impl::chooseSfzFile()
     SharedPointer<CNewFileSelector> fs = owned(CNewFileSelector::create(frame_));
 
     fs->setTitle("Load SFZ file");
-    fs->setDefaultExtension(CFileExtension("SFZ", "sfz"));
 
     // also add extensions of importable files
+    fs->addFileExtension(CFileExtension("SFZ", "sfz"));
```

Comment:
SFZ is not among "importable files" as indicated in the comment.
Can you move it back up a few lines?